### DISCUSSION TITLE
Unable to screenshot in osu!lazer whilst fullscreen.

### DISCUSSION THREAD

**Discussion by vijarsan** [ppy/osu#14827] [Q&A] [answered]
I've tried to screenshot on osu!lazer in fullscreen and it would never work, either it would show a small resolution version of the main menu or would just be black.

I'm using ShareX on Windows 11 preview, and I haven't had an issue with screenshotting in other fullscreen games.

https://user-images.githubusercontent.com/44067130/134150001-20ea4540-eef3-4480-9ac5-8278a0c058fa.mp4

**Comment by peppy** [comment, marked as answer]
I think this is the same thing reported here? https://github.com/ppy/osu/issues/14546

**Reply by vijarsan** [reply]
Oh, yeah I believe so, I had the same issue with his screenshot preview having a tiny resolution aswell.

Apologies for the duplicate question I didn't see that someone else had the same issue.
